global.json

```diff
@@ -1,8 +0,0 @@
-{
-  "projects": [
-    "src"
```

Comment:
Not automagically. You have to manually change 'PackageReference' <=> 'ProjectReference'. But the auto project/package interchangability feature is being discussed: https://github.com/dotnet/roslyn-project-system/issues/199